presto-native-execution/src/test/java/com/facebook/presto/nativeworker/NativeQueryRunnerUtils.java

```diff
@@ -33,6 +33,7 @@ public static Map<String, String> getNativeWorkerSystemProperties()
     {
         return ImmutableMap.<String, String>builder()
                 .put("optimizer.optimize-hash-generation", "false")
+                .put("optimizer.use-mark-distinct", "false")
```

Comment:
Prestissimo now supports MarkDistinct Plannode. So why is this optimizer hint needed inspite of that ?

Reply:
@aditi-pandit This was added before the change. Removed after rebase